src/test/groovy/edu/ie3/datamodel/utils/validation/ConnectorValidationUtilsTest.groovy

```diff
@@ -0,0 +1,259 @@
+/*
+ * © 2021. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+package edu.ie3.datamodel.utils.validation
+
+import static edu.ie3.datamodel.models.StandardUnits.*
+import static edu.ie3.util.quantities.PowerSystemUnits.*
+
+import org.locationtech.jts.geom.Point
+import edu.ie3.datamodel.models.input.connector.LineInput
+import edu.ie3.datamodel.models.input.connector.type.Transformer2WTypeInput
+import edu.ie3.datamodel.models.input.connector.type.Transformer3WTypeInput
+import edu.ie3.datamodel.models.input.system.characteristic.OlmCharacteristicInput
+import edu.ie3.datamodel.models.voltagelevels.GermanVoltageLevelUtils
+import org.locationtech.jts.geom.LineString
+import tech.units.indriya.ComparableQuantity
+
+import javax.measure.quantity.Angle
+import javax.measure.quantity.Dimensionless
+import javax.measure.quantity.ElectricConductance
+import javax.measure.quantity.ElectricPotential
+import javax.measure.quantity.ElectricResistance
+import javax.measure.quantity.Power
+
+import edu.ie3.datamodel.exceptions.InvalidEntityException
+import edu.ie3.test.common.GridTestData
+import edu.ie3.util.geo.GeoUtils
+import org.locationtech.jts.geom.Coordinate
+import spock.lang.Specification
+import tech.units.indriya.quantity.Quantities
+
+class ConnectorValidationUtilsTest extends Specification {
+
+	def "Smoke Test: Correct line throws no exception"() {
+		given:
+		def line = GridTestData.lineFtoG
+
+		when:
+		ValidationUtils.check(line)
+
+		then:
+		noExceptionThrown()
+	}
+
+	private static final Point testCoordinate = GeoUtils.DEFAULT_GEOMETRY_FACTORY.createPoint(new Coordinate(10, 10))
+	private static final LineInput invalidLineLengthNotMatchingCoordinateDistances = new LineInput(
```

Comment:
```suggestion
	defPoint testCoordinate = GeoUtils.DEFAULT_GEOMETRY_FACTORY.createPoint(new Coordinate(10, 10))
	def LineInput invalidLineLengthNotMatchingCoordinateDistances = new LineInput(
```

I prefer using `def` for declaration. Java-style declaration does weirdo things to the scope of the variable.